SCClassLibrary/JITLib/Patterns/Pdef.sc

```diff
@@ -538,10 +538,12 @@ EventPatternProxy : TaskProxy {
 					]).asStream
 				}
 			}{
+				var fadeOutCleanup = cleanup.copy;
```

Comment:
just one little thing, please move the variable declaration out of the if statement. It makes the code less effiient. Also it will cause warnings if you have `Post Inline Warnings` on. Generally, it is good to keep this option ticked whenever you work on the common class library for that reason. I know it is not so pleasing aesthetically!

Reply:
Sure, since `constrainStream` already puts 11 slots on the stack, the 12th being unconditionally allocated even when no fade is used probably isn't any noticeable space saving... I'm not familiar with the SC inlining process, but for my education, if an `if` branch can't be inlined, does that make the other branch not inlined either?

Reply:
So, I've implemented the requested change, i.e. moved the var up top... Just sayin',  in case the commit below wasn't noticed